electron/images_cache.ts

```diff
@@ -0,0 +1,34 @@
+import { existsSync, createWriteStream, mkdirSync } from 'graceful-fs'
```

Comment:
Would be good if we already do a strucutre for this.
To many files in `electron` makes things confusing.

What do you think about `electron/common/imagecache` or `electron/imagecache` or `electron/image/cache` ?

Reply:
electron already creates a Cache directory, maybe use that one.
`heroic/Cache/images`

Reply:
and I believe you can do something like:
`const imagesCachePath = app.getPath('cache')/images`

Reply:
I'll use the current heroic/Cache :+1:

Reply:
oh I see what you mean @Nocccer , you don't mean the cache, you directory, you mean where I puth the images_cache.ts file

Reply:
I agree that we need some convention, I put the file there because there are other files there and adding one new folder for just one small file felt weird to me, but I think we need to find some pattern to use to put this accessory files